packages/ui-dom/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@emcasa/ui-dom",
+  "version": "0.1.0",
+  "description": "EmCasa's UI library",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "author": "EmCasa <dev@emcasa.com>",
+  "scripts": {
+    "build": "babel src -d dist",
+    "watch": "yarn build --watch",
+    "test": "jest",
+    "prepublish": "yarn build"
+  },
+  "dependencies": {
+    "@emcasa/theme": "^0.1.0",
+    "@rebass/grid": "6.0.0-4",
+    "react": "16.5.2",
+    "react-dom": "16.5.2",
+    "styled-components": "3.4.9",
+    "styled-icons": "3.0.0",
+    "styled-system": "3.1.6"
+  },
```

Comment:
Is there any way we can keep common dependencies such as `styled-components`, `react,`styled-system` and `styled-icons` in a single place?

Also, IMO `react` and `react-dom` should be peer dependencies.